File: C/2455.c
int averageValue(int* nums, int numsSize) {
    
    int i, sum=0, count=0;


    sum = 0, count = 0;
    for (i = 0; i < numsSize; i++) {
        if ((nums[i] % 2) == 0) {
            if ((nums[i] % 3) == 0) {
                sum += nums[i];
                count++;
            }
        }
    }

    if(count==0)
      return 0;

    return sum/count;  
    
}

File: Python/2455.py
class Solution(object):
    def averageValue(self, nums):
        sum =0
        count = 0
        for num in nums:
            if(num % 2 == 0 and num % 3 == 0):
                sum += num
                count += 1
        if count == 0:
            return 0
        else:
            return sum/count       
                
        """
        :type nums: List[int]
        :rtype: int
        """
        